src/main/scala/gemmini/GemminiConfigs.scala

```diff
@@ -187,9 +186,11 @@ case class GemminiArrayConfig[T <: Data : Arithmetic, U <: Data, V <: Data](
   //==========================================================================
   // cisc-gemmini miscellaneous constants (some redundant with above)
   //==========================================================================
-  val rob_entries      = reservation_station_full_entries + reservation_station_partial_entries
+  val rob_entries      = pow(2, log2Up(reservation_station_entries_per_type)).toInt * 3
```

Comment:
Why not just make this `reservation_station_entries_per_type * 3`?

Reply:
Is now the sum of the number of entries for the different types